### PR TITLE
Add concurrency support by heroku example

### PR DESCRIPTION
**Pull request by abtris**
- https://github.com/heroku-examples/node-concurrency/blob/master/server.js

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,208 +1,223 @@
...
-    if (process.env.DEBUG) {
-      console.log('Unauthorized access by %s', auth.name);
-    }
-  }
-});
 
-app.post('/', function (req, res) {
-  if(req.body !== undefined) {
-    req.body.pipe(through(line => processLine(line, req.prefix, req.defaultTags)));
+if (require.main === module) {
```
Comment (kuba-kubula) [inline app.js:7]: what is this here for @abtris ?

Comment (abtris) [inline app.js:7]: Try run npm test without this.

File: app.js [hunk 2]
```diff
@@ -1,208 +1,223 @@
...
-let statsd = new StatsD(parseStatsdUrl(process.env.STATSD_URL));
-let app = module.exports = express();
-let allowedApps = loadAllowedAppsFromEnv();
-
-if (process.env.DEBUG) {
-  console.log('Allowed apps', allowedApps);
-  statsd.send = wrap(statsd.send.bind(statsd), console.log.bind(null, 'Intercepted: statsd.send(%s):'));
-}
+let throng = require('throng');
+let WORKERS = process.env.WEB_CONCURRENCY || 1;
```
Comment (kuba-kubula) [inline app.js]: please use `parseInt(process.env.WEB_CONCURRENCY || 1, 10)`;

File: app.js [hunk 3]
```diff
@@ -1,208 +1,223 @@
...
+let port = process.env.PORT || 3000;
```
Comment (kuba-kubula) [inline app.js]: dtto as above

**Comment by kuba-kubula** [comment]
Otherwise looks good to me

**Comment by abtris** [comment]
ok, I fix parseInt

**Comment by abtris** [comment]
Done

**Pull request merged**